validator/sawtooth_validator/journal/block_validator.py

```diff
@@ -169,11 +104,6 @@ def stop(self):
 
 
 class BlockValidator(object):
-    """
```

Comment:
Is doc going to be restored in a future PR?

Reply:
Yeah, I am planning to break this class up further.